h2o-r/h2o-package/R/frame.R

```diff
@@ -1793,6 +1794,7 @@ h2o.quantile <- function(x,
 }
 
 #' @rdname h2o.quantile
+#' @importFrom utils capture.output
```

Comment:
this is technically redundant as we already declared it once, but on the other hand it nicely document that this function uses external package, so no harm to leave it